mkdocs.yml

```diff
@@ -3,17 +3,19 @@ site_name: tinygrad docs
 site_url: https://docs.tinygrad.org/
 repo_url: https://github.com/tinygrad/tinygrad/
 nav:
-- Home: index.md
-- Quickstart: quickstart.md
-- MNIST Tutorial: mnist.md
-- API:
-  - Tensor: tensor.md
-  - Function: function.md
-  - dtypes: dtypes.md
-  - nn (Neural Networks): nn.md
-- Showcase: showcase.md
-- Developer: developer.md
-- Environment: env_vars.md
+- Home:
+  - index.md
+  - Quickstart: quickstart.md
+  - Showcase: showcase.md
+  - MNIST Tutorial: mnist.md
+  - API Reference:
+    - Tensor: tensor.md
+    - Function: function.md
+    - dtypes: dtypes.md
+    - nn (Neural Networks): nn.md
+  - Notes:
```

Comment:
Notes is meh... env var should be in API and developer can have its own section?

Reply:
yea i didn't really know where to put these two